lib/msal-core/src/ServerRequestParameters.ts

```diff
@@ -113,7 +113,7 @@ export class ServerRequestParameters {
         queryParameters = this.addHintParameters(account, queryParameters);
 
         // sanity check for developer passed extraQueryParameters
-        const eQParams: StringDict = request.extraQueryParameters;
+        const eQParams: StringDict|null = request?.extraQueryParameters;
```

Comment:
This could be a one line fix, correct? Something like:

```js
const eQParams: StringDict = request && request.extraQueryParameters;
```

We definitely want to turn on strict null checks and fix the types, but isn't need to fix this bug, correct?

Reply:
Correct.

Reply:
Looks like the TS version here is too old for `?.`.